library/build.gradle.kts

```diff
@@ -24,6 +24,10 @@ kotlin {
     macosX64()
     macosArm64()
 
+    compilerOptions {
```

Comment:
Does this has to be in upstream as well? 🤔 Maybe worth to document in the readme as long as it is experimental? 🤔